docs/book/v6/tutorials/create-book-module.md

```diff
@@ -642,78 +735,46 @@ declare(strict_types=1);
-use Api\App\ConfigProvider as AppConfigProvider;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\ContainerInterface;
+use Psr\Container\NotFoundExceptionInterface;
 
-class ConfigProvider
+class RoutesDelegator
 {
-    public function __invoke(): array
+    /**
+     * @throws ContainerExceptionInterface
+     * @throws NotFoundExceptionInterface
+     */
+    public function __invoke(ContainerInterface $container, string $serviceName, callable $callback): Application
     {
-        return [
-            'dependencies'     => $this->getDependencies(),
-            'doctrine'     => $this->getDoctrineConfig(),
-            MetadataMap::class => $this->getHalConfig(),
-        ];
-    }
+        $uuid = ConfigProvider::REGEXP_UUID;
 
-    private function getDependencies(): array
-    {
-        return [
-            'delegators' => [
-                Application::class => [
-                    RoutesDelegator::class
-                ]
-            ],
-            'factories' => [
-                BookHandler::class    => AttributedServiceFactory::class,
-                BookService::class    => AttributedServiceFactory::class,
-                BookRepository::class => AttributedRepositoryFactory::class,
-            ],
-            'aliases'   => [
-                BookServiceInterface::class     => BookService::class,
-            ],
-        ];
-    }
+        /** @var RouteCollectorInterface $routeCollector */
+        $routeCollector = $container->get(RouteCollectorInterface::class);
 
-    private function getDoctrineConfig(): array
-    {
-        return [
-            'driver' => [
-                'orm_default'   => [
-                    'drivers' => [
-                        'Api\Book\Entity' => 'BookEntities'
-                    ],
-                ],
-                'BookEntities'  => [
-                    'class' => AttributeDriver::class,
-                    'cache' => 'array',
-                    'paths' => __DIR__ . '/Entity',
-                ],
-            ],
-        ];
-    }
+        $routeCollector->group('/book')
+            ->post('', PostBookHandler::class, 'book::create-book');
 
-    private function getHalConfig(): array
-    {
-        return [
-            AppConfigProvider::getCollection(BookCollection::class, 'books.list', 'books'),
-            AppConfigProvider::getResource(Book::class, 'book.show')
-        ];
-    }
+        $routeCollector->group('/book/' . $uuid)
+            ->get('', GetBookHandler::class, 'book::view-book');
+
+        $routeCollector->group('/books')
+            ->get('', GetBookCollectionHandler::class, 'book::list-books');
 
+        return $callback();
+    }
 }
 ```
 
+We need to configure access to the newly created endpoints, add `books::list-books`, `book::view-book` and `book::create-book` to the authorization rbac array, under the `UserRole::ROLE_GUEST` key.
+> Make sure you read and understand the rbac documentation.
```

Comment:
Add link to the documentation mentioned in this sentence so devs can find it easily:
https://docs.dotkernel.org/dot-rbac-guard/v4/configuration/